test/models/location_test.rb

```diff
@@ -1,6 +1,62 @@
 require 'test_helper'
 require 'models/shared/taxonomies_base_test'
+require 'rfauxfactory'
 
 class LocationTest < ActiveSupport::TestCase
   include TaxonomiesBaseTest
+  valid_loc_data_list = [
```

Comment:
similar test for orgs called this `valid_org_name_list`, if this is new pattern in our tests, could you also name it similarly? I guess `valid_loc_name_list` would be better fit